templates/meta_woz/dialogues/templates.yaml

```diff
@@ -0,0 +1,111 @@
+dataset: meta_woz
+subset: dialogues
+templates:
+  4b64c6e9-0aa0-431f-85b5-8367daa4773a: !Template
+    id: 4b64c6e9-0aa0-431f-85b5-8367daa4773a
+    jinja: "What does this conversation between a chatbot and a client talk about\
+      \ ? \n{{turns | join(',  ')}}\n|||\n{{domain.replace('_', ' ') | lower }}"
+    name: predict_domain_2
+    reference: ''
+    task_template: false
+  56151136-82a4-455b-98ed-aea6ee1c273d: !Template
+    id: 56151136-82a4-455b-98ed-aea6ee1c273d
+    jinja: '{% set count = namespace(value=0) %}
+
+      {% for i in range(range(turns|length) | random() - 1) %}
+
+      {{["AI Assistant", "Client"][i% 2]}}: {{turns[i]}}
+
+      {% set count.value= i + 1 %}
+
+      {% endfor %}
+
+      {{["AI Assistant", "Client"][ count.value % 2]}}:
+
+      |||
+
+      {{turns[count.value] }}
+
+
+      '
+    name: predict_random
+    reference: ''
+    task_template: true
+  5e90f705-9d63-4917-acc9-3baabc6ee5e9: !Template
+    id: 5e90f705-9d63-4917-acc9-3baabc6ee5e9
+    jinja: "{% set rand_index= namespace(value=range(turns|length)|random()) %}\n\
+      {% for utterance in turns %}\n{% if loop.index0 == rand_index.value %}\n{{[\"\
+      Chatbot\", \"Human\"][loop.index0 % 2]}}: \n{% else %}\n{{[\"Chatbot\", \"Human\"\
+      ][loop.index0 % 2]}}: {{utterance}}\n{% endif %}\n{% endfor %}\n|||\n{{turns[rand_index.value]\
+      \ }}"
+    name: 'predict_missing '
+    reference: ''
+    task_template: false
+  77a9e854-08ef-4f2d-86f9-ed077f18b39d: !Template
+    id: 77a9e854-08ef-4f2d-86f9-ed077f18b39d
+    jinja: "This conversation is between an AI assistant and a human. What is the\
+      \ human inquiring about ? \n{{turns | join(',  ')}}\n|||\n{{domain.replace('_',\
+      \ ' ') | lower }}"
+    name: predict_domain_3
+    reference: ''
+    task_template: false
+  9150f1e0-8b9e-4f24-bc58-1cbb230cb8d9: !Template
+    id: 9150f1e0-8b9e-4f24-bc58-1cbb230cb8d9
+    jinja: "This conversation is between an AI assistant and a human. what does it\
+      \ talk about ? \n{{turns | join(',  ')}}\n|||\n{{domain.replace('_', ' ') |\
+      \ lower }}"
+    name: predict_domain_1
+    reference: ''
+    task_template: false
+  98b0697f-3144-40b6-b6ae-5d57c3f08db3: !Template
+    id: 98b0697f-3144-40b6-b6ae-5d57c3f08db3
+    jinja: "Chatbot: {{turns[0]}}\nHuman: {{turns[1]}}\nChatbot: \n|||\n{{turns[2]}}"
+    name: predict_next_stmt_1
+    reference: ''
+    task_template: true
+  a02a980d-630f-4845-8b4d-b0eb072110e9: !Template
+    id: a02a980d-630f-4845-8b4d-b0eb072110e9
+    jinja: "{% if turns|length % 2 == 1 %}\n{% for i in range((turns|length) // 2)\
+      \ %}\nBot: {{turns[2*i]}}\nUser: {{turns[2*i+1]}}\n{% endfor %}\nBot: \n|||\n\
```

Comment:
It looks like the lines are turning out like
```
Bot: Hello how may I help you? User: I need some event help
Bot: Certainly, help with an event. How can I assist you? User: Can you please get me some tickets
```
Can you debug so they appear on new lines?